internal/repos/github.go

```diff
@@ -695,24 +693,19 @@ func (s *GithubSource) AffiliatedRepositories(ctx context.Context) ([]types.Code
 			"retryAfter", retry,
 		)
 	}()
-	out := make([]types.CodeHostRepository, 0, len(repos))
-	for !done {
+	out := make([]types.CodeHostRepository, 0)
```

Comment:
```suggestion
	out := make([]types.CodeHostRepository, 0, 100)
```

nit: This could save multiple allocation/expansion of the slice when getting the first page. But I have no statistic to back my suggestion, so just a nit. 100 is the page size of the GitHub v3 API.

Reply:
I'd say for now we don't need this optimisation. There's so many other allocations and io in the same code path it probably won't make a difference.

Good spot though 😄